manticore/core/cpu/arm.py

```diff
@@ -293,7 +293,7 @@ class Armv7LinuxSyscallAbi(SyscallAbi):
 
     # EABI standards:
     #  syscall # is in R7
-    #  arguments are passed in R0-R6
+    #  arguments are passed in R0-R5
```

Comment:
The code below (`for i in range(6): yield 'R'.format(i)` returns exactly 6 args while `R0-R6` would be 7. Anyway afaik syscalls have maximum of 6 args.